assets/company/logo-fandom-powered-by-wikia-two-lines.svg

```diff
@@ -1,33 +1,45 @@
-    <path id="logo-fandom-powered-by-wikia-two-lines-path-2" d="M30.603.169H0v31.912h30.603z"/>
-    <linearGradient x1="33.016%" y1="84.794%" x2="84.225%" y2="35.32%" id="logo-fandom-powered-by-wikia-two-lines-gradient">
-      <stop stop-color="#ADED00" offset="0%"/>
-      <stop stop-color="#00D6D6" offset="100%"/>
-    </linearGradient>
-    <path id="logo-fandom-powered-by-wikia-two-lines-path-3" d="M6.913.149H.076v9.688h6.837V.149z"/>
-  </defs>
-  <g fill="none" fill-rule="evenodd">
-    <path d="M48.283 1.884c-5.14 0-8.594 3.466-8.594 8.624v16.63h4.285V14.915h4.497v-4.07h-4.497v-.37c0-2.912 1.516-4.389 4.505-4.389.292 0 .571.05.867.104.112.02.226.04.342.058l.216.034v-4.21l-.163-.021a19.47 19.47 0 0 1-.383-.057 6.92 6.92 0 0 0-1.075-.11zm14.72 16.828c0 2.958-1.433 4.521-4.146 4.521-3.72 0-4.276-2.833-4.276-4.521 0-2.874 1.559-4.522 4.276-4.522 2.653 0 4.01 1.482 4.146 4.522m4.304.846a36.71 36.71 0 0 0-.018-.854c-.202-5.294-3.432-8.583-8.432-8.583-2.58 0-4.763.869-6.314 2.513-1.45 1.536-2.247 3.683-2.247 6.045 0 4.916 3.484 8.623 8.105 8.623 2.197 0 3.98-.839 5.186-2.432a11.6 11.6 0 0 0 .737 2.157l.05.11h4.32l-.118-.267c-1.206-2.715-1.244-5.481-1.27-7.312m11.612-9.438c-2.464 0-4.547.868-6.025 2.509-1.384 1.537-2.146 3.698-2.146 6.083v8.426h4.285v-8.426c0-2.874 1.416-4.521 3.886-4.521 2.469 0 3.885 1.647 3.885 4.52v8.427h4.285v-8.426c0-2.385-.762-4.546-2.146-6.083-1.478-1.64-3.56-2.508-6.024-2.508m23.299 8.591c0 2.958-1.433 4.521-4.145 4.521-3.721 0-4.277-2.833-4.277-4.521 0-2.874 1.56-4.522 4.277-4.522 2.653 0 4.009 1.482 4.145 4.522m3.684 0V4.276h-4.133v6.8c-1.317-.595-2.283-.955-3.92-.955-5.055 0-8.45 3.44-8.45 8.558 0 4.916 3.508 8.623 8.161 8.623 2.37 0 4.08-.715 5.213-2.185.181.78.447 1.619.763 2.401l.047.118h4.324l-.116-.266c-1.386-3.185-1.89-6.923-1.89-8.658m15.426 0c0 2.873-1.535 4.521-4.21 4.521-2.677 0-4.212-1.648-4.212-4.521 0-2.874 1.535-4.522 4.211-4.522s4.211 1.648 4.211 4.522m-4.21-8.591c-5.003 0-8.497 3.533-8.497 8.59 0 5.059 3.494 8.591 8.496 8.591 5.003 0 8.497-3.532 8.497-8.59s-3.494-8.591-8.497-8.591" fill="#FFF"/>
-    <g transform="translate(127.537 10.105)">
-      <mask id="logo-fandom-powered-by-wikia-two-lines-mask-1" fill="#fff">
-        <use xlink:href="#logo-fandom-powered-by-wikia-two-lines-path-1"/>
-      </mask>
-      <path d="M24.028 2.343C22.668.821 20.748.016 18.476.016c-2.314 0-4.21 1.106-5.377 3.125C11.933 1.122 10.036.016 7.723.016 5.451.016 3.53.82 2.17 2.343.902 3.763.204 5.753.204 7.947v9.086H4.49V7.947c0-2.454 1.178-3.862 3.233-3.862s3.233 1.408 3.233 3.862v9.086h4.286V7.947c0-2.454 1.179-3.862 3.234-3.862 2.054 0 3.233 1.408 3.233 3.862v9.086h4.286V7.947c0-2.193-.699-4.184-1.967-5.604" fill="#FFF" mask="url(#logo-fandom-powered-by-wikia-two-lines-mask-1)"/>
+<svg width="184px" height="44px" viewBox="0 0 184 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Please use width and height without `px` to be consistent with the rest of the files.
Also, run `npm run svgo` to optimize the source files. There is a lot of ids, transforms, paths and other unnecessary cruft here.